src/pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.targets

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Target Name="EnsureNETCoreAppRuntime" 
+          Condition="'$(RuntimeIdentifier)' != '' AND '$(EnsureNETCoreAppRuntime)' != 'false'" 
+          AfterTargets="RunResolvePackageDependencies">
```

Comment:
Runs after RunResolvePackageDependencies since that has produces PackageDependencies item which I'm reading and has the right condition to avoid running in design time.